src/d4bl/observability/langfuse.py

```
@@ -57,57 +83,61 @@ def initialize_langfuse():
+        logger.debug(
+            "Install with: pip install langfuse openinference-instrumentation-crewai"
+        )
+        _langfuse_init_state = False
         return None
     except Exception as e:
-        print(f"⚠️ Error initializing Langfuse: {e}")
-        import traceback
-
-        traceback.print_exc()
-        _langfuse_initialized = False
+        logger.error("Error initializing Langfuse: %s", e, exc_info=True)
```

Ruff (0.15.2)

[warning] Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)